decidim-proposals/app/controllers/decidim/proposals/proposals_controller.rb

```diff
@@ -97,37 +96,15 @@ def create
         end
       end
 
-      def compare
-        enforce_permission_to :edit, :proposal, proposal: @proposal
-        @step = STEP2
-        @similar_proposals ||= Decidim::Proposals::SimilarProposals
```

Comment:
Found a leftover to this variable: https://github.com/decidim/decidim/blob/d6fdc054ef009831ed9c7337b1aca9645d8f160a/decidim-proposals/app/views/decidim/proposals/proposals/_wizard_header.html.erb#L4
Can you remove that line 🙏🏽 ?

Reply:
Line removed